src/connectors/search-box/__tests__/connectSearchBox-test.js

```diff
@@ -103,6 +103,49 @@ describe('connectSearchBox', () => {
     }
   });
 
+  it('provides a function to clear the query and perform new search', () => {
+    const rendering = sinon.stub();
+    const makeWidget = connectSearchBox(rendering);
+
+    const widget = makeWidget();
+
+    const helper = jsHelper(fakeClient);
+    helper.search = sinon.stub();
+
+    widget.init({
+      helper,
+      state: helper.state,
+      createURL: () => '#',
+      onHistoryChange: () => {},
+    });
+
+    { // first rendering
+      expect(helper.state.query).toBe('');
+      const renderOptions = rendering.lastCall.args[0];
+      const {refine} = renderOptions;
+      refine('bip');
+      expect(helper.state.query).toBe('bip');
+      expect(helper.search.callCount).toBe(1);
+    }
+
+    widget.render({
+      results: new SearchResults(helper.state, [{}]),
+      state: helper.state,
+      helper,
+      createURL: () => '#',
+    });
+
+    { // Second rendering
+      expect(helper.state.query).toBe('bip');
+      const renderOptions = rendering.lastCall.args[0];
+      const {clear, query} = renderOptions;
+      expect(query).toBe('bip');
+      clear();
```

Comment:
Refine with a new query after a clear? Or test that `helper.search()` is called?